leadership.html

```diff
@@ -420,95 +407,84 @@ <h1>رهبری</h1>
             <ul>
-              <li>Facilitates discussions <span class="highlight">across teams</span>, ensuring that everyone has an
-                opportunity to share their opinion and be heard,
-                and that discussion outcomes tie to stated goals.
-                <br /><span class="highlight">Ensures relevant parties are included in discussions.</span>
-                <br /><span class="highlight">Guides discussions toward decisions, clarifies and gets buy-in.</span>
-                <br /><span class="highlight">Ensures decisions are communicated to stakeholders</span>
+              <li>بحث‌ها را <span class="highlight">درون تیم‌ها</span>، تسهیل می‌کند و اطمینان حاصل می‌کند همه فرصت به اشتراک گذاشتن نطرات خود و شنیده شدن را دارند
+                و اینکه نتایج بحث با اهداف بیان شده مرتبط است.
+                <br /><span class="highlight">اطمینان حاصل می‌کند که طرف‌های مربوطه در بحث‌ها شرکت می‌کنند.</span>
+                <br /><span class="highlight">
+                  بحث‌ها را به سمت تصمیم گیری هدایت می‌کند، شفاف سازی می‌کند و ؟؟.
+                  Guides discussions toward decisions, clarifies and gets buy-in.</span>
+                <br /><span class="highlight">اطمینان حاصل می‌کند که تصمیمات به ذی‌نفعان اطلاع رسانی می‌شود.</span>
               </li>
             </ul>
           </div>
 
           <div class="decision">
-            <span class="pill">Decision Making</span>
+            <span class="pill">تصمیم‌گیری</span>
             <ul>
               <li>
-                Takes ownership of decisions made <span class="highlight">across teams</span> by helping them make clear
-                decisions in alignment with organisational goals,
-                backing decisions made, and taking responsibility for their success.
-                <br /><span class="highlight">Solicits input from team but makes decisions where necessary, even if
-                  consensus can’t be reached</span>
-                <br />Helps create the team's roadmap for a <span class="highlight">group of teams</span> in line with
-                the company and technology vision.
-                <br />Ensures accountability is practiced <span class="highlight">throughout those teams</span>.
+                <li> با کمک به آنها در اتخاذ تصمیمات شفاف در راستای اهداف سازمان، حمایت از تصمیمات اتخاذ شده و بر عهده گرفتن مسئولیت موفقیت آنها، مالکیت تصمیمات گرفته شده در <span class="highlight">درون تیم‌ها</span> را به عهده می گیرد.
+                <br /><span class="highlight">از تیم درخواست می‌کند، اما در صورت لزوم تصمیم می‌گیرد، حتی اگر توافق حاصل نشود</span>
+                <br />به ایجاد نقشه راه تیم برای <span class="highlight">گروهی از تیم‌ها</span> در راستای چشم انداز شرکت و فناوری کمک می‌کند.
+                <br />اطمینان حاصل می‌کند که مسئولیت‌پذیری در <span class="highlight">سرتاسر تیم‌ها</span> انجام می‌شود.
                 Demonstrates these behaviours themselves.
```

Comment:
same as [this comment](https://github.com/AbbasiMohamad/software-engineer-skills-matrix/pull/3/files#r1083525510)

Reply:
the same as [this](https://github.com/AbbasiMohamad/software-engineer-skills-matrix/pull/3#discussion_r1083541711)